lib/codeowners/checker.rb

```diff
@@ -108,7 +105,7 @@ def main_group
     end
 
     def consistent?
-      check!.values.all?(&:empty?)
+      results.values.all?(&:empty?)
```

Comment:
we should add `UnrecognizedLine` list to the results - so existing errors in the file also trigger failure? // @jonatas

Reply:
good point. we need to consider unrecognized lines but I'm not sure about the severity of it. should we adopt some configuration for it?

If we don't have any unrecognized line in our use cases, I guess we can adopt it as the "green" state.